execute_pipeline.py

```diff
@@ -145,6 +145,8 @@ def _load_config_spec(config_spec, config_sections, repl_vars):
         all_config_data = yaml.load(config_file)
     for section in config_sections:
         data.update(all_config_data[section])
+    if language in all_config_data:
```

Comment:
Is it a problem to always add the language? (Will someone have reason to leave it out for a file which contains a section for that language, if they are actually processing that language?) 

Also, is it a problem to load the language section twice? Since an update is being done here, it seems like the result would be idempotent. I can see the argument for avoiding the re-update for clarity, though.